OpenRA.Game/Traits/Health.cs

```diff
@@ -44,7 +44,7 @@ public Health(ActorInitializer init, HealthInfo info)
 			MaxHP = info.HP;
 
 			hp = init.Contains<HealthInit>() ? init.Get<HealthInit, int>() * MaxHP / 100 : MaxHP;
-			if (hp <= 0)
+			if (hp < 0)
```

Comment:
This was the most important part of #7475, and changing it will reintroduce #7426.